phpunit.xml.dist

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
```

Comment:
it prefer to use the schema from the vendor dir

Reply:
Should we update it in package template also?

https://github.com/yiisoft/package-template/blob/master/phpunit.xml.dist

Reply:
may be

Reply:
Yes. Definitely.